### ISSUE TITLE
About the performance on ImageNet-100

### ISSUE THREAD

**Issue by seekingup** [SsnL/moco#2]
Hi, thanks for your inspiring work!

I'm a bit confused about the Reference Validation Accuracy on ImageNet-100. 
In this table, the accuracy is much lower when batch_size=256 and lr=0.03. However, batch_size=256 and lr=0.03 should be the optimal hyper-parameters for the original MoCo.

I notice the accuracy for MoCo v2 67.52% on ImageNet-100 is the same with ImageNet (67.5%). Is there a mistake, or ImageNet-100 does require different hyper-parameters?

Hope for your reply!

![image](https://user-images.githubusercontent.com/40528686/112778498-42bb2100-9077-11eb-8c19-c5154471c898.png)

**Comment by ssnl** [comment]
Hi @seekingup , sorry that I missed this issue. I went back and double checked the numbers from my checkpoints. They are indeed correct. It might be possible that ImageNet-100 has a different set of optimal hyperparameters. It might also be useful to know that these are the numbers I obtained after the full linear classifier training, which is often <1.5% worse than picking the best performing one throughout training. However, the relative comparison trend usually holds.
